paper.tex

```diff
@@ -1100,11 +1099,24 @@ \section*{Precision of ancestry information}
 and (D) \relate~\citep{speidel2019method}
 for 11 samples of a 2.7kb region of the \textit{Drosophila melanogaster} ADH
 locus~\citep{kreitman1983nucleotide}. The \argweaver\ example
-is a sample from the inferred posterior distribution.
+is a sample from the inferred posterior distribution. In each ARG,
+colours are assigned to edges based on the time of the edge's child node: 
+lighter colours are assigned to edges with older child
+nodes, darker colours are assigned to edges with younger child nodes
 Note that the position of points on the
-Y axis is that chosen for graphical clarity by the graphviz \texttt{dot}
-algorithm,  rather than being
-a direct measure of time. Additional notes about these ARGs.
+Y axis is chosen for graphical clarity by the graphviz \texttt{dot}
+algorithm, rather than as a direct measure of time.
+Line width corresponds to the genomic span of the edge. Node colour
+refers to the number of parents of the node: red is zero, brown is one, 
+green to blue are two to N parents. The brightness
+of the node is determined by the genomic span over which
+the node has two or more children divided by the span where it has any
+children. Edges are visualised below each ARG, where each row is an edge 
```

Comment:
Perhaps "The genomic intervals associated with edges are visualised..." here? We're trying to make an edge mean the graph-theoretic thing for this paper, rather than the tskit interpretation.